apps/voxelize.cpp

```diff
@@ -211,11 +190,18 @@ int main( int argc, char* argv[] )
           "             (defaults: 0.1 for Compartments and Somas,\n"
           "                        1.5 / duration for Spikes\n"
           "                        1.0 for Synapses and VSD)\n"
+          "- inputMin/inputMax: minimum and maximum input values to be\n"
+          "                     considered for rescaling\n"
+          "                     (defaults: [0.0, 2.0] for Spikes and Synapses\n"
+          "                                [-190.0, 0.0] for Compartments with TestData, [-80.0, 0.0] otherwise\n"
+          "                                [-15.0, 0.0] for Somas with TestData, [-80.0, 0.0] otherwise\n"
```

Comment:
Sure, they can. But in my tests (LFP, compartment and soma reports, etc) I rarely saw it, and specially not with the current TestData. That's why I didn't want to include a wider range for the defaults, because then in the output the significant data would be reduced to a very narrow range, compared to the total output range.